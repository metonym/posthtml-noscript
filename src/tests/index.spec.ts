import posthtml from 'posthtml';
import { noscript } from '../';

describe('posthtmlNoscript', () => {
  test('noscript matches snapshot', () => {
    posthtml()
      .use(
        noscript({ content: 'You need to enable JavaScript to run this app.' })
      )
      .process(
        `<body>
          <div id="root"></div>
        </body>`
      )
      .then((result: { html: string }) => {
        expect(result.html).toMatchSnapshot();
      });
  });
});
