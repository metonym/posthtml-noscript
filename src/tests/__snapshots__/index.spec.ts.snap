// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`posthtmlNoscript noscript matches snapshot 1`] = `
"<body>
          <div id=\\"root\\"></div>
        <noscript>You need to enable JavaScript to run this app.</noscript></body>"
`;
